cross/xc16.txt

```diff
@@ -0,0 +1,24 @@
+# This file assumes that path to the Microchip xc16 toolchain is added
+# to the environment(PATH) variable, so that Meson can find
+# xc16-gcc and xc16-ar while building.
+[binaries]
+c = 'xc16-gcc'
+ar = 'xc16-ar'
+strip = 'xc16-gcc'
+
+[host_machine]
+system = 'bare metal'
+cpu_family = 'xc16'
+cpu = 'dspic'
```

Comment:
Based on the discussion, should this not instead be `cpu_family` dspic and `cpu` something more specific?

Reply:
updated this